codetransformer/decompiler.py

```diff
@@ -107,6 +115,29 @@ def _process_jump(instr, queue, stack, body, context):
         )
 
 
+# @_process_instr.register(instrs.JUMP_IF_TRUE_OR_POP)
```

Comment:
This was an aborted implementation that I was looking at for reference.  Removed.